op-conductor/rpc/execution_proxy.go

```diff
@@ -38,3 +39,15 @@ func (api *ExecutionProxyBackend) GetBlockByNumber(ctx context.Context, number r
 	}
 	return result, nil
 }
+
+func (api *ExecutionProxyBackend) SetMaxDASize(ctx context.Context, maxTxSize hexutil.Big, maxBlockSize hexutil.Big) bool {
```

Comment:
Should function be called on ExecutionMinerProxyBackend instead of ExecutionProxyBackend

Reply:
oh actually it should be deleted, will fix